src/utils/directories.py

```diff
@@ -32,22 +31,34 @@
 
 # == Sprites dirs ==
 BLOCKS_DIR = SPRITES_DIR + "blocks/"
+CASTLE_DIR = SPRITES_DIR + "castle/"
+FLAG_DIR = SPRITES_DIR + "flag/"
+BACKGROUND_DIR = SPRITES_DIR + "background/"
 ENEMIES_DIR = SPRITES_DIR + "enemies/"
 ICONS_DIR = SPRITES_DIR + "icons/"
 PLAYERS_DIR = SPRITES_DIR + "players/"
 POWER_UPS_DIR = SPRITES_DIR + "power_ups/"
 
-# == Icons ==
-# == icons name ==
-COIN_ICON_DIR = ICONS_DIR + "coin/"
-PASTEL_ICON_DIR = ICONS_DIR + "pastel/"
+# == Flag ==
+# == Flag name ==
+FLAG_STAND_DIR = FLAG_DIR + "stand/"
+FLAG_PIPE_DIR = FLAG_DIR + "pipe/"
+FLAG_WIN_DIR = FLAG_DIR + "win/"
+FLAG_SUPPORT_DIR = FLAG_DIR + "support/"
+
+# == Background ==
+# == Background name ==
+BUSH_BACKGROUND_DIR = BACKGROUND_DIR + "bush/"
+CLOUD_BACKGROUND_DIR = BACKGROUND_DIR + "cloud/"
+MOUNTAIN_BACKGROUND_DIR = BACKGROUND_DIR + "mountain/"
+TREE_BACKGROUND_DIR = BACKGROUND_DIR + "tree/"
 
 # == Blocks ==
 # == blocks name ==
 BRICK_DIR = BLOCKS_DIR + "brick/"
 BROKE_BLOCK_DIR = BLOCKS_DIR + "broke_block/"
 COIN_DIR = BLOCKS_DIR + "coin/"
-MASTERY_BLOCK_DIR = BLOCKS_DIR + "mastery_block/"
+MYSTERY_BLOCK_DIR = BLOCKS_DIR + "mistery_block/"
```

Comment:
Unnecessary change

Reply:
Mastery block in Spanish is 'Bloqueo de dominio' only changed to a more representative name